packages/react/src/components/Card/CardToolbar.jsx

```diff
@@ -210,6 +213,8 @@ const CardToolbar = ({
           <DateTimePicker
             id={testId}
             i18n={mergedI18n}
+            dateTimeMask="L HH:mm"
```

Comment:
Should this mask be a configurable prop also? Secondly, should its default match the current "YYYY-MM-DD HH:mm" format so that the existing titles/snapshots aren't changed?

Reply:
sure, this is a good point

Reply:
Waiting on these changes to merge. Otherwise we should be good to go.